src/pages/StocktakeEditPage.js

```diff
@@ -90,8 +90,8 @@ export class StocktakeEditPage extends React.Component {
       case 'countedTotalQuantity':
         return {
           type: isEditable ? 'editable' : 'text',
-          cellContents: item.hasBatchWithQuantityChange ? item.countedTotalQuantity : '',
-          placeholder: tableStrings.no_change,
+          cellContents: item.hasCountedBatches ? item.countedTotalQuantity : '',
```

Comment:
Minor suggestion: stocktakeItem rather than item, just to be consistent with the other methods.

Reply:
good point